resources/qml/EncryptionIndicator.qml

```diff
@@ -41,7 +41,7 @@ Image {
         case Crypto.TOFU:
             return qsTr("Encrypted by an unverified device, but you have trusted that user so far.");
         case Crypto.MessageUnverified:
-            return qsTr("Key is from an untrusted source like forwarded from another user or the online key backup. For this reason we can't verify who sent the message.");
```

Comment:
So, my english sucks, but my assumption was that wording it like this means basically something like "Key is from an untrusted source. Examples of that could be, that the key was forwarded by another user or that the key is downloaded from the online key backup". I think changing this to "likely" actually loses that meaning somewhat? @LorenDB or @redsky17 , could you check what makes more sense?

Reply:
I agree with what @redsky17 said. I'd personally go with something like "Key is from an untrusted source, possibly forwarded from another user or the online key backup. For this reason we can't verify who sent the message.".